packages/ember-routing-views/lib/views/link.js

```diff
@@ -408,19 +409,11 @@ var LinkComponent = EmberComponent.extend({
       queryParams = {};
     }
 
-    if (attrs.disabledClass) {
-      this.set('disabledClass', attrs.disabledClass);
-    }
-
-    if (attrs.activeClass) {
-      this.set('activeClass', attrs.activeClass);
-    }
-
     if (attrs.disabledWhen) {
-      this.set('disabled', attrs.disabledWhen);
+      this.set('disabled', getValue(attrs.disabledWhen));
     }
 
-    var currentWhen = attrs['current-when'];
+    var currentWhen = getValue(attrs['current-when']);
 
     if (attrs.currentWhen) {
```

Comment:
I thought the same thing but didn't act because of the deprecation and this being 1.13 and all